internal/provider/apisix/provider.go

```diff
@@ -80,15 +82,20 @@ func New(updater status.Updater, readier readiness.ReadinessManager, opts ...pro
 	}
 
 	return &apisixProvider{
-		client:     cli,
-		Options:    o,
-		translator: &translator.Translator{},
-		updater:    updater,
-		readier:    readier,
-		syncCh:     make(chan struct{}, 1),
+		client:           cli,
+		adcDebugProvider: common.NewADCDebugProvider(cli.Store, cli.ConfigManager),
```

Comment:
Why does it belong to the provider rather than the ADC client?

Reply:
Makes sense. move it.